69.md

```diff
@@ -0,0 +1,96 @@
+NIP-69
+======
+
+Linked Crytographic Identities
+------------------------------
+
+`draft` `optional`
+
+Nostr users may have other crytographic identities such as PGP or SSL certificates. This NIP introduces a replaceable event of kind `10069` that links these identities with a Nostr pubkey.
+
+The event MAY contain zero or more `i` tags. Each tag MUST have, in the following order:
+ - A type (listed below) and a public key fingerprint that MUST NOT contain spaces, joined by a `:`
+ - A signature of the message (described below) in base64 format, unwrapped
+ - The full public key in base64 format, unwrapped
+
+Types: `pgp`, `x509`
```

Comment:
Should keys be enumerated? As drafted, this NIP can work for any key that can produce a signature.

Reply:
Not listing them would lead to undesired inconsistencies (example, someone prefixes the `i` tag with `gpg:` instead of `pgp:`).

We can add a nostr type but there again it's important to define the exact format of the event to be signed.

Reply:
Why would it be a problem if a client prefixed the `i` tag with `gpg` for a `gpg` signature?

Reply:
So clients can easily find these events?

In my particular case I am indexing all sorts of release artifacts (APKs, DMGs, AppImages, etc) some signed with certificates and others with PGP. Given an artifact, I want to be able to quickly look up if its signer also controls a nostr key. If we use `pgp` or `gpg` interchangeably now clients have to make two different requests. Don't you think it makes sense?

Of course this is nostr and people can ultimately publish whatever they want

Reply:
Not necessarily. You: "I support PGP proof", Them: "I have GPG proof", You: "..."

> If we use pgp or gpg interchangeably now clients have to make two different requests.

While this may be superfluous since your case seems to flag encryption support, if you wanted to be extra safe you could do `{ "i#": { 'pgp:<key>', 'gpg:<key>'}` ... `'pgp:<key>' or 'gpg:<key>'` in a single `REQ`.

> Of course this is nostr and people can ultimately publish whatever they want

And it is because of this I questioned enumeration.

Reply:
I think it's good to have some basic agreement on these labels! I will change it to "recommended" types so it's just that, a recommendation, and let's add a `nostr` one.